docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Deploying the LSP on Polygon
+sidebar_label: Deploying the LSP on Polygon
+---
+UMA's infrastructure is also available on Polygon for developers to make use of as a scaling solution. The following will explain the details on how to deploy on Polygon. 
+
+Deploying to Polygon follows a near-identical launch process as to launching any UMA contract on Ethereum. Using one of our launch repo's and amending the destination network, you can quickly deploy your contracts on Polygon.
+
+## How UMA works on Polygon
```

Comment:
This is great content! I think this section works better in `Understanding UMA` however, since you are giving a broad overview of how our polygon architecture works. Thoughts on separating out the `How UMA Works on Polygon` portion into its own page in `Understanding UMA`, and then only having the deployment tutorial here?

Reply:
Happy to move it across if we feel we need to, although I am not sure we are going to be keeping the `Understanding UMA` or am I wrong to assume that?

Reply:
The name may change, but we'll always at least have an introductory "content explanation" section. We've always tried to keep the developer section geared towards immediately actionable tutorials etc, rather than explaining concepts here.